include/King.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "Piece.h"
+
+class King : public Piece {
+public:
+    using Piece::Piece;
+
+    const bool isValidMove(const std::pair<std::pair<int, int>, std::pair<int, int>>&, const std::vector<std::vector<std::unique_ptr<Piece>>>&) override;
+
+private:
+};
```

Comment:
Follow the same recommendations as mentioned for the Bishop and Piece headers.

Reply:
Done.